joy_body_ik_node:
  ros__parameters:
    axis_linear:
      x: 1
      y: 0
      z: 3
    axis_angular:
      roll: 6
      pitch: 7
      yaw: 4
    lock: 2
    reset: 3
